### PR TITLE
`epoch` --> `tick` replace

### PR DESCRIPTION
**Pull request by jhellerstein**

### CODE REVIEW

File: book/example_4_neighbors.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ Finally we print the neighbor vertices as follows:
 ```rust,ignore
     my_join -> unique() -> for_each(|n| println!("Reached: {}", n));
 ```
-The [unique](./surface_ops.gen.md#unique) operator removes duplicates from the stream to make things more readable. Note that `unique` does not run in a streaming fashion, which we will talk about more [below](#strata-and-epochs).
+The [unique](./surface_ops.gen.md#unique) operator removes duplicates from the stream to make things more readable. Note that `unique` does not run in a streaming fashion, which we will talk about more [below](#strata-and-ticks).
```
Comment (MingweiSamuel) [inline book/example_4_neighbors.md:139]: Was here before but only noticing now - this doesn't link to anything

File: book/todo.md [hunk 2]
```diff
@@ -79,16 +79,16 @@
...
+    -  The [`run_tick()`](https://hydro-project.github.io/hydroflow/doc/hydroflow/scheduled/graph/struct.Hydroflow.html#method.run_tick), [`run_stratum()`](https://hydro-project.github.io/hydroflow/doc/hydroflow/scheduled/graph/struct.Hydroflow.html#method.run_stratum), [`run()`](https://hydro-project.github.io/hydroflow/doc/hydroflow/scheduled/graph/struct.Hydroflow.html#method.run), and [`run_async()`](https://hydro-project.github.io/hydroflow/doc/hydroflow/scheduled/graph/struct.Hydroflow.html#method.run_async) methods provide other ways to control the graph execution.
     - Also `run_available()` `next_stratum()` and `recv_events` are important
 - More generally, document the **Core API**
 
-- **Strata and Epochs** -- explain the concept of strata and epochs, and how they relate to the `run_epoch()` and `run_stratum()` methods.
+- **Strata and ticks** -- explain the concept of strata and ticks, and how they relate to the `run_tick()` and `run_stratum()` methods.
```
Comment (MingweiSamuel) [inline book/todo.md:86]: title needs capitalized T

**Review by MingweiSamuel — APPROVED** [review]
just a couple tiny nit changes

File: book/example_4_neighbors.md (hunk 1 above)
Comment (jhellerstein) [reply book/example_4_neighbors.md:139]: reinstated thanks.

**Pull request merged**